FBTweak/FBTweak.h

```diff
@@ -86,6 +86,12 @@ typedef id FBTweakValue;
  */
 @property (readonly ,nonatomic) dispatch_block_t currentValue;
 
+/**
+ @abstract \c YES if the tweak is enabled, \c NO otherwise. Default is \c YES. If \c NO, the
+ \c currentValue block will not be called.
+ */
```

Comment:
1. I wouldn't say the the default is YES in the protocol.
2. I think it should say something like `If \c NO the block is not allowed to be called.`.  Then we want to enforce this, we'll have to wrap the block in `FBActionTweak` to assert when the block is called and `enabled` is `NO`.

Reply:
Move it to the class

Reply:
The classes and protocols are a mess there:
https://github.com/Lightricks/Tweaks/blob/master/FBTweak/_FBTweakCollectionViewController.m#L53
It checks the protocol conformance of the tweak for deciding which cell to create.  `_FBActionTweakTableViewCell` is the only cell that holds a class and not a protocol.

Reply:
yes it's a mess

Reply:
I've decided to change the cell so it holds the protocol